lib/wavefront-direct-reporter.js

```diff
@@ -128,28 +128,35 @@ class WavefrontDirectReporter extends ScheduledReporter {
   }
 
   appendCounter(counter, points) {
-    let tags = helper.tagger(this.registry.getTags(counter.name), this.globaltags);
-    points.push(formatter.counterPoint(counter, this.prefix, '', tags));
+    const nameAndTags = this.getDecodedKey(counter);
```

Comment:
consider the following, which programmatically generates the methods instead of writing them by hand:
```
const funs = {
  appendCounter: "counterPoint",
  appendGauge: "gaugePoint",
  appendMeter: "meterPoints",
  appendTimer: "timerPoints",
  appendHistogram: "histoPoints"
}

Object.keys(funs).forEach((key) => {
  WavefrontDirectReporter.prototype[key] = function(metric, points) {
    const [name, tags] = this.getDecodedKey(metric);
    const newPoints = formatter[funs[key]](metric, name, this.prefix, '', tags);
    const params = Array.isArray(newPoints) ? newPoints : [newPoints];
    points.push(...params);
  }
  return acc;
});
```